### PR TITLE
Enrollment server onboarding

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: powerauth-backend-tests/pom.xml [hunk 1]
```diff
@@ -86,6 +86,11 @@
             <artifactId>powerauth-restful-security-spring</artifactId>
             <version>1.3.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>com.wultra.security</groupId>
+            <artifactId>enrollment-server-api-model</artifactId>
+            <version>1.3.0-SNAPSHOT</version>
```
Comment (banterCZ) [inline powerauth-backend-tests/pom.xml:92]: It should be 1.4.0-SNAPSHOT now.

File: powerauth-webflow-tests/pom.xml [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
     <artifactId>powerauth-webflow-tests</artifactId>
-    <version>1.0.0</version>
+    <version>1.2.0</version>
     <packaging>jar</packaging>
 
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.3.7.RELEASE</version>
+        <version>2.6.6</version>
```
Comment (banterCZ) [inline powerauth-webflow-tests/pom.xml:18]: Ideally it should be 2.6.8

File: powerauth-backend-tests/pom.xml [hunk 3]
```diff
@@ -86,6 +86,11 @@
             <artifactId>powerauth-restful-security-spring</artifactId>
             <version>1.3.0-SNAPSHOT</version>
```
Comment (banterCZ) [inline powerauth-backend-tests/pom.xml:87]: Shouldn't this version be 1.4.0?

File: powerauth-backend-tests/pom.xml (hunk 1 above)
Comment (romanstrobl) [reply powerauth-backend-tests/pom.xml:92]: Yes.

File: powerauth-backend-tests/pom.xml (hunk 3 above)
Comment (romanstrobl) [reply powerauth-backend-tests/pom.xml:87]: Yes, we should switch to `1.4.0-SNAPSHOT`.

**Review by romanstrobl — CHANGES_REQUESTED** [review]
Let's update the version to `1.4.0-SNAPSHOT` and merge it.

**Review by romanstrobl — APPROVED** [review]
Perfect thanks. We can finalize the Web Flow UI tests revival another day -- the `soap` suffix in URL is incorrect, but there will be other problems, too.

**Pull request merged**